packages/amplify-provider-awscloudformation/lib/initialize-env.js

```diff
@@ -9,13 +9,10 @@ function run(context, providerMetadata) {
   }
 
   const zipFilename = '#current-cloud-backend.zip';
-  const backendDir = context.amplify.pathManager.getBackendDirPath();
-  const tempDir = `${backendDir}/.temp`;
+  const amplifyDir = context.amplify.pathManager.getAmplifyDirPath();
+  const tempDir = `${amplifyDir}/.temp`;
```

Comment:
May be we should use [temp](https://www.npmjs.com/package/temp) module to generate temporary files and content instead of creating one in the amplify directory

Reply:
Yeah, we had a discussion regarding this a while back of using amplify dir vs using the users systems local tmp dir and decided to use the amplify dir due to variation in every OS's tmp/ dir location.

Reply:
the module I linked handles all of that. Need not be now, but we should use this later just keep our directories clean